package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "git-blame",
   "main": "./lib/index",
-  "version": "1.7.0",
+  "version": "1.8.0",
```

Comment:
sorry i didnt see this before. my tooling handles the updating of the package version. please remove this change